utils/spacewalk-common-channels.ini

```
@@ -2917,36 +2917,36 @@ name     = AlmaLinux 8 (%(arch)s)
 gpgkey_url = https://repo.almalinux.org/almalinux/RPM-GPG-KEY-AlmaLinux
 gpgkey_id = 3ABB34F8
 gpgkey_fingerprint = 5E9B 8F56 17B5 066C E920 57C3 488F CF7C 3ABB 34F8
-repo_url = https://repo.almalinux.org/almalinux/8/BaseOS/%(arch)s/os/
+repo_url = https://mirrors.almalinux.org/mirrorlist/8/baseos
```

For coherence, I'd add the trailing slash:

```suggestion
repo_url = https://mirrors.almalinux.org/mirrorlist/8/baseos/
```
Same for the other lines